code/modules/organs/subtypes/parasite.dm

```diff
@@ -1,3 +1,7 @@
+#define INFECTION_SPEED_QUICK 35
+#define INFECTION_SPEED_NORMAL 20
+#define INFECTION_SPEED_SLOW 15
```

Comment:
I dont see the advantage of using defines here, when two of them are used only once and another one not at all.